app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/auth/AuthNavigation.kt

```diff
@@ -89,6 +91,21 @@ fun NavGraphBuilder.authGraph(
             onNavigateToLanding = {
                 navController.popBackStack(route = LANDING_ROUTE, inclusive = false)
             },
+            onNavigateToPasswordGuidance = {
+                navController.navigateToMasterPasswordGuidance()
+            },
+            onNavigateToPreventAccountLockout = {
+                navController.navigateToPreventAccountLockout()
+            },
+            onNavigateToOnboarding = {
```

Comment:
@david-livefront Was unsure on this, I know that we are supposed to be in an "unlocked state" here is this how to handle?

Reply:
The onboarding flow should probably be handled via state-based navigation in the `RootNavViewModel`